src/utils/components/ColumnManagementModal/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const MAX_VIEW_COLS = 9;
+
+export const NAME_COLUMN_ID = 'name';
+export const READ_ONLY_COLUMNS = new Set([NAME_COLUMN_ID]);
```

Comment:
if its a set and not a string const , name should be as any var ur are using in the app